visidata/loaders/hdf5.py

```diff
@@ -42,10 +42,16 @@ def iterload(self):
 
 
     def openRow(self, row):
+        import h5py
         if isinstance(row, BaseSheet):
             return row
-        if isinstance(row, h5py.Object):
-            return H5ObjSheet(row)
+        if isinstance(row, h5py.HLObject):
```

Comment:
What is the difference between `h5py.Object` and `h5py.HLObject`?

Reply:
It looks like, with the most recent version of `h5py`, `h5py.Object` is no more:
```
>>> import h5py
>>> dir(h5py)
['AttributeManager', 'Dataset', 'Datatype', 'Empty', 'ExternalLink', 'File', 'Group', 'HLObject', 'HardLink', 'Reference', 'RegionReference', 'SoftLink', 'VirtualLayout', 'VirtualSource', '_ModuleWrapper', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '__version__', '_conv', '_errors', '_hl', '_objects', '_proxy', '_register_converters', '_register_lzf', '_warn', 'absolute_import', 'check_dtype', 'check_enum_dtype', 'check_ref_dtype', 'check_string_dtype', 'check_vlen_dtype', 'defs', 'enable_ipython_completer', 'enum_dtype', 'filters', 'get_config', 'get_enum', 'get_vlen', 'h5', 'h5a', 'h5ac', 'h5d', 'h5ds', 'h5f', 'h5fd', 'h5g', 'h5i', 'h5l', 'h5o', 'h5p', 'h5pl', 'h5py_warnings', 'h5r', 'h5s', 'h5t', 'h5z', 'highlevel', 'is_hdf5', 'new_enum', 'new_vlen', 'ref_dtype', 'regionref_dtype', 'register_driver', 'registered_drivers', 'run_tests', 'special_dtype', 'string_dtype', 'unregister_driver', 'utils', 'version', 'vlen_dtype']
```

Using `h5py.HLObject` here was just my best guess and it worked! ;]

Reply:
Gotcha. So, this will croak with earlier versions of `h5py`?

Reply:
It looks like there has never been a `h5py.Object`.
```
git clone https://github.com/h5py/h5py
cd h5py
git log -p | grep 'class Object' | grep -v ObjectID | grep -v ObjectHeader
```

![image](https://user-images.githubusercontent.com/453543/96200685-c9ad8e80-0f5a-11eb-8a39-0656d6d8f608.png)

-- https://geekandpoke.typepad.com/geekandpoke/2010/10/geekpoke-looks-behind-the-scenes-of-coders.html